inc/L4/Epoch/EpochQueue.h

```diff
@@ -38,8 +38,10 @@ struct EpochQueue {
   using SharableLock = TSharableLock;
   using ExclusiveLock = TExclusiveLock;
   using RefCount = std::atomic<std::uint32_t>;
-  using RefCounts = Interprocess::Container::
-      Vector<RefCount, typename Allocator::template rebind<RefCount>::other>;
+  // @jerinphilip: dirty-fix, browsermt/L4 hopefully does not need IPC.
```

Comment:
What is IPC? Replacing interprocess::vector with vector will lead to a race condition. THe questions is, do we care that we have a race condition.

Reply:
I wrote it short for inter-process-communication. @kpu told me at some point it's okay and just offsets, where offsets become 0 or so when not in a multiproc setting (our use-case). I haven't been able to find much documentation on this.

Reply:
i guess it's fine then. Tests pass, it's not obviously broken? xD